website/source/docs/agent/configuration/cloud-auto-join.html.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: "docs"
+page_title: "Cloud Auto-join"
+sidebar_current: "docs-agent-cloud-auto-join"
+description: |-
+  Nomad supports automatic cluster joining using cloud metadata on various providers.
+---
+
+# Cloud Auto-joining
```

Comment:
There should be a call out somewhere that this is only a subset of the cloud providers and for the most up to date documentation see the library itself

Reply:
Maybe use the yellow here: https://www.nomadproject.io/guides/namespaces.html